controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -1228,4 +1330,10 @@ private TxnStatus handleDataNotFoundException(Throwable ex) {
     abstract CompletableFuture<Data<T>> getRetentionSet();
 
     abstract CompletableFuture<Void> updateRetentionSet(Data<T> retention);
+
+    abstract CompletableFuture<Void> createEpochTransitionNode(byte[] epochTransition);
```

Comment:
I'm not seeing this call being exercized directly in any test case.

Reply:
yes its not exercised directly from test.. we have a unit test for StreamMetadataStore.. 
StreamMetadataStore uses Stream objects (its abstract implementation is PersisentStreambase whose implementations are ZkStream or InMemoryStream).
 
So we test both flavours of streamMetadataStore.type (zk and inMemory) in our various unit tests which calls all these methods.

Reply:
i have added a new test scenario where i am manipulating the start scale flow..  i am calling this method directly from that test case now..